### PR TITLE
Invite user to participant team api should be idempotent.

### PR DESCRIPTION
**Pull request by gauthamzz**
#256 is solved. @TroJan is this okay ?

### CODE REVIEW

File: apps/participants/serializers.py [hunk 1]
```diff
@@ -43,6 +43,6 @@ def validate_email(self, value):
 
     def save(self):
         email = self.validated_data.get('email')
-        return Participant.objects.create(user=User.objects.get(email=email),
-                                          status=Participant.ACCEPTED,
-                                          team=self.participant_team)
+        return Participant.objects.get_or_create(user=User.objects.get(email=email),
+                                                 status=Participant.ACCEPTED,
+                                                 team=self.participant_team)
```
Comment (deshraj) [inline apps/participants/serializers.py]: @gauthamzz can you please remove the extra spaces that you have added in line 47 and 48?

Comment (gauthamzz) [inline apps/participants/serializers.py]: But that's for PEP8 (tabs over spaces) . should i remove them . then travis shows error

Comment (deshraj) [inline apps/participants/serializers.py]: We prefer using spaces all over the codebase. So, please make sure that you also use spaces. :-)

If it contains spaces, then it's fine.

Comment (gauthamzz) [inline apps/participants/serializers.py]: so is there any need to change anything as of now . @deshraj just put those to align them correctly

**Comment by deshraj** [comment]
I would prefer to remove the spaces in line 47 and 48 but it's just a minor change. You don't need to change it and we are good to go for now. I will merge it. 

Thanks :-)

**Pull request merged**